endToEndTests/test/invalidQueries/insertionContains_invalidPattern.json

```diff
@@ -0,0 +1,18 @@
+{
+  "testCaseName": "Insertion Contains with invalid pattern",
+  "query": {
+    "action": {
+      "type": "Aggregated"
+    },
+    "filterExpression": {
+      "type": "InsertionContains",
+      "column": "insertions",
+      "position": 25701,
+      "value": "CC+++"
+    }
+  },
+  "expectedError": {
+    "error": "Bad request",
+    "message": "The field 'value' in the InsertionContains expression does not contain a valid regex pattern: \"CC+++\""
```

Comment:
I would be nice to tell the user what a valid pattern would be.

Reply:
I adapted it. As a side note, since this message is user-facing (I guess?), the regex for zero or any nucleotide sequence differs here (user '?' vs internal '.*')

Reply:
Why does it differ? Where does the `?` come in? I didn't see it in the code?

Reply:
With this PR I would say that SILO already gets regex and rejects everything else. So we do not do that transformation here but I would expect it already in LAPIS.
We can change this later, but for now it is out of scope